exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,13 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (danielsRole = "mentor") {
+  console.log(`Hi, I'm ${name}, I'm a ${danielsRole}.`);
+} else if (danielsRole = "student") {
+    console.log(`Hi, I'm ${name}, I'm a student.`);
+}
+
```

Comment:
nice use of template literals here! on line 12 you are assigning danielsRole to "mentor" and again on line 14 you are assigning danielsRole to "student", if you are wanting to make a comparison you would use strict equality like `if (danielsRole === "mentor") `